client/src/components/settings/Portfolio.tsx

```diff
@@ -112,25 +122,26 @@ class PortfolioSettings extends Component {
     return isEqual(original, edited);
   };
 
-  isFormValid = id => {
-    const { portfolio } = this.state;
-    const toValidate = find(portfolio, createFindById(id));
-    if (!toValidate) {
-      return false;
-    }
-    const { title, url, image, description } = toValidate;
+  // TODO: Check if this function is required or not
+  // isFormValid = id => {
```

Comment:
Is this function required as it isn't used anywhere and it looks similar to the function above it namely `isFormPristine`

Reply:
For now, I've left this function commented out.